config/main.py

```diff
@@ -998,6 +1012,8 @@ def cli_sroute_to_config(ctx, command_str, strict_nh = True):
         if 'prefix' in prefix_str and 'vrf' in prefix_str:
             # prefix_str: ['prefix', 'vrf', Vrf-name, ip]
             vrf_name = prefix_str[2]
+            if not is_vrf_exists(config_db, vrf_name):
```

Comment:
Do we have check to ensure vrf is removed only after all interface bindings are removed? Otherwise this is only a check in the add-path.

Reply:
This is only a check in the add path.

Reply:
In fact the current vrf removal logic removes all these bindings implicitly using the function del_interface_bind_to_vrf